apache/superset
# chore: Update font-sizes in DatabaseModal

### SUMMARY
This PR removes hacky usages of font-sizes by replacing them with the exact font-sizes available in the theme config.

### BEFORE

https://user-images.githubusercontent.com/60598000/162433958-816ec03a-e060-4912-a4a5-8bb523b62260.mp4

### AFTER

https://user-images.githubusercontent.com/60598000/162434015-b50c06a9-d521-4d89-9cf4-36960d80a9df.mp4

### TESTING INSTRUCTIONS
1. Open the Add / Edit Database modal
2. Make sure the font-sizes look balanced

### ADDITIONAL INFORMATION
